releases/release-1.14/release_team.md

```diff
@@ -0,0 +1,13 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |  |
+| Enhancements | Claire Laurence ([@claurence](https://github.com/claurence)) | |
+| CI Signal | Maria Ntalla ([@mariantalla](https://github.com/mariantalla)) | |
+| Test Infra | Amit Watve ([@amwat](https://github.com/amwat)) | |
+| Bug Triage | | |
+| Branch Manager | Hannes Hoerl ([@hoegaarden](https://github.com/hoegaarden)) | |
+| Docs | Jim Angel ([@jimangel](https://github.com/jimangel)) |  |
+| Release Notes | Dave Strebel ([@dstrebel](https://github.com/dstrebel)) |  |
+| Communications | Natasha Woods ([@nwoods3](https://github.com/nwoods3)) |  Kaitlyn Barnard ([@kbarnard10](https://github.com/kbarnard10)) |
+| Patch Release Managers |  | |
+| Approval Notifier | k8s-ci-robot ([@k8s-ci-robot](https://github.com/k8s-ci-robot)) | |
```

Comment:
I have a stupid question...is `Approval Notifier` needed? Why do we add it here?

Reply:
Good point, @nikhita! Removed.